CHANGES.rst

```diff
@@ -1,11 +1,22 @@
 Changelog
 =========
 
-1.0.1 (unreleased)
+1.0.0 - beta
```

Comment:
Since there is no public version in pypi for `senaite.sync` yet, I suggest to replace `1.0.0 -  beta` by:

```
1.0.0 (unreleased)
------------------
```